src/components/field/svg/FieldGeneratedLines.tsx

```diff
@@ -13,13 +13,17 @@ class FieldPathLines extends Component<Props, State> {
   declare context: React.ContextType<typeof DocumentManagerContext>;
 
   render() {
+    const path = this.context.model.document.pathlist.activePath;
     let generatedPathString = "";
-    const generated = this.context.model.document.pathlist.activePath.generated;
-    this.context.model.document.pathlist.activePath.generated.forEach(
-      (point) => {
-        generatedPathString += `${point.x},${point.y} `;
-      }
-    );
+    const trajectory = path.generating
+      ? path.generationProgress
+      : path.generated;
+    // preserve this so that it updates when mutating the in-progress trajectory in place
+    //eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const iteration = path.generationIterationNumber;
```

Comment:
It's needed because we mutate the generationProgress array in place for performance reasons, so I needed to depend on something else that will change on progress updates. I chose the iteration number which (for now) doesn't necessarily match the solver, but is incremented on every progress update. Later we can get the iteration number from the solver and perhaps show the value somewhere, but it will still be usable for this purpose.

Reply:
Can we use the convention `_` instead of iteration, since the field you are getting is fairly self-documenting..

Reply:
Also, we should leave a comment to future developers that may try to remove it in the future.